packages/zowe-explorer-api/src/vscode/ZoweVsCodeExtension.ts

```diff
@@ -136,20 +141,11 @@ export class ZoweVsCodeExtension {
         return undefined;
     }
 
-    public static async inputBox(inputBoxOptions: vscode.InputBoxOptions): Promise<string> {
-        if (!inputBoxOptions.validateInput) {
-            // adding this for the theia breaking changes with input boxes
-            // eslint-disable-next-line @typescript-eslint/explicit-function-return-type
-            inputBoxOptions.validateInput = (value) => null;
-        }
-        return await vscode.window.showInputBox(inputBoxOptions);
-    }
-
     private static async saveCredentials(profile: imperative.IProfileLoaded): Promise<boolean> {
         let save = false;
         const saveButton = "Save Credentials";
         const message = `Save entered credentials in plain text for future use with profile ${profile.name}?\nSaving credentials will update the local information file.`;
-        await vscode.window.showInformationMessage(message, { modal: true }, ...[saveButton]).then((selection) => {
+        await Gui.showMessage(message, { items: [saveButton], vsCodeOpts: { modal: true } }).then((selection) => {
```

Comment:
And I think this is why I'm asking. 😋 
Why not just use `void` here too if we don't plan to use the value (which should be `undefined`)?

I'm not suggesting we change the use of the `void` operator.
These questions are just to help me (and hopefully others) understand this a bit better.

Reply:
Primarily because `await` is exclusive to `async` functions, it is more of a shorthand here to dismiss the Promise instead of wrapping each call in a `.then()` and `.catch()` - honestly not keen about the use of `void` but was just adapting its current use in the codebase to the new `Gui` calls.